test/unit/user_test.rb

```diff
@@ -66,11 +66,14 @@ class UserTest < ActiveSupport::TestCase
     end
   end
 
+
+  ## NOTE:  We manage allowed email addresses dynamically now, but schools.nyc.gov
+  # is to always be allowed, and it's OK not to test registration success for the others.
   [generate_email_without_valid_domain].each do |new_email|
     test 'should not save user without schools.nyc.gov domain in email' do
       @user.email = "testing@gmail.com"
       @user.save
-      assert_equal([" should end in @schools.nyc.gov"], @user.errors[:email])
+      assert_equal([" should end in @schools.nyc.gov or another participating school address"], @user.errors[:email])
```

Comment:
Should you also create a test with a second domain other than `schools.nyc.gov`?

Reply:
Maybe?  What would you want the test to be testing for?  Maybe a success on inserting user with non DOE address?  (The user insert with DOE address is currently going by same mechanism as the non-DOE would, though...)

Reply:
You could test creating an AllowedUserEmailMask and see if a user can register with that.